cpp/include/cudf/utilities/type_dispatcher.hpp

```diff
@@ -82,6 +82,39 @@ struct id_to_type_impl {
 template <cudf::type_id Id>
 using id_to_type = typename id_to_type_impl<Id>::type;
 
+/**
+ * @brief "Returns" the corresponding type that is stored on the device when using `cudf::column`
+ *
```

Comment:
Maybe give an example of how `decimal32` is stored as a column of `int32_t`.

Reply:
Done.